lib/improver/utilities/temporal.py

```diff
@@ -223,26 +223,42 @@ def iris_time_to_datetime(time_coord):
     Returns:
         list of datetime.datetime objects
             The time element(s) recast as a python datetime object.
-
     """
     time_coord.convert_units('seconds since 1970-01-01 00:00:00')
     return [datetime.utcfromtimestamp(value) for value in time_coord.points]
 
 
-def dt_to_utc_hours(dt_in):
+def datetime_to_iris_time(dt_in, time_units="hours"):
     """
-    Convert python datetime.datetime into hours since 1970-01-01 00Z.
+    Convert python datetime.datetime into hours, minutes or seconds
+    since 1970-01-01 00Z.
 
     Args:
         dt_in (datetime.datetime object):
             Time to be converted.
+
+        time_units (str):
+            Name of time unit. Currently only "hours", "minutes" or
+            "seconds" are supported. Alternatively, an origin time can be
+            supported, for example "seconds since 1970-01-01 00:00:00".
```

Comment:
At the moment, the "since 1970-01-01 00:00:00" part of the units is ignored. I don't mind how you fix this, whether you add something to the docstring or if you change it so people can't put in "since ..." type units

Reply:
I've added a comment to the docstring.